Project.toml

```
@@ -16,12 +16,14 @@ MacroTools = "0.5"
 Reexport = "0.2, 1.0"
 Requires = "1"
 URIParser = "0.4"
+URIs = "1.1"
 julia = "1"
 
 [extras]
 Glob = "c27321d9-0574-5035-807b-f59d2c89b15c"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 URIParser = "30578b45-9adc-5946-b283-645ec420af67"
+URIs = "5c2747f8-b7ea-4ff2-ba2e-563bfd36b1d4"
 
 [targets]
-test = ["Glob", "Test", "URIParser"]
+test = ["Glob", "Test", "URIs"]
```

We probably still want to test on URIParser if possible.

```suggestion
test = ["Glob", "Test", "URIParser", "URIs"]
```